public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# http://www.robotstxt.org
+User-agent: *
+Disallow:
```

Comment:
Do we need this `robots.txt` file at all? [From the site you linked](http://www.robotstxt.org/robotstxt.html):

> To allow all robots complete access
> ```
> User-agent: *
> Disallow:
> ```
>  (or just create an empty "/robots.txt" file, or don't use one at all)